### PR TITLE
Add bitwarden_rs

### PR DESCRIPTION
**Pull request by ArmaanT**
This PR adds bitwarden_rs for all the password-managing goodness

### CODE REVIEW

**Comment by ArmaanT** [comment]
There wasn't an existing helm chart for helm 3, which is why I ended up using helm on my own. Is there a significant advantage to not using helm for something like this?

**Comment by pawalt** [comment]
Not strictly. I frankly don't care either way. It's just that if you're trying to understand this deploy, it's easier to understand 3 kube manifests than it is to understand a full helm chart. Again, this isn't a problem with this deploy since you can treat it as a normal web application, but things that interact w/ the kube api or have complex layouts may be a different story.

File: bitwarden/values.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+replicaCount: 1
+
+image:
+  repository: bitwardenrs/server-mysql
+  tag: latest
```
Comment (pawalt) [inline bitwarden/values.yml]: Please pin this

Comment (pawalt) [reply bitwarden/values.yml]: Just use whatever bitwarden version is on latest right now

Comment (ArmaanT) [reply bitwarden/values.yml]: Pinned

**Pull request merged**